tests/e2e/helpers.go

```diff
@@ -203,6 +225,15 @@ func (c *mcTestCtx) ensureE2EPool() {
 	c.trackPool(pool)
 }
 
+// This will creat a new machine config sub pool with one random node from worker pool to speed up the test.
```

Comment:
nit: create*

Looks like the actual spelling error came from line 219 above though (we can fix that separately).